common/src/main/resources/data/forge/tags/items/crafting_tables.json

```diff
@@ -0,0 +1,7 @@
+{
+    "replace": false,
+    "values": [
+      "minecraft:crafting_table",
+      "#blue_skies:crafting_tables"
```

Comment:
should not contain other mods by as this will break without blue skies, the other mods should be required to add their tables to such a tag

Reply:
You could add

"forge:conditions": [{
        "type": "forge:mod_loaded",
        "modid": "blue_skies"
      }],